src/test/java/org/dice/factcheckbenchmark/component/DummySystemAdapter.java

```diff
@@ -0,0 +1,96 @@
+package org.dice.factcheckbenchmark.component;
+
+import org.dice.factcheckbenchmark.system.api.Client;
+import org.dice.factcheckbenchmark.system.api.FactCheckHobbitResponse;
+import org.hobbit.core.components.AbstractSystemAdapter;
+import org.hobbit.sdk.JenaKeyValue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import java.io.IOException;
+
+
+public class DummySystemAdapter extends AbstractSystemAdapter {
+    private static final Logger logger = LoggerFactory.getLogger(DummySystemAdapter.class);
+    private static JenaKeyValue parameters;
+    private String databaseContainer;
+    private String factcheckContainer;
+    private String factcheckContainerUrl;
+
+    @Override
+    public void init() throws Exception {
+        super.init();
+        logger.debug("Init()");
+        // Your initialization code comes here...
+
+        parameters = new JenaKeyValue.Builder().buildFrom(systemParamModel);
+        logger.debug("SystemModel: " + parameters.encodeToString());
+        // You can access the RDF model this.systemParamModel to retrieve meta data about this system adapter
+
+
+    }
+
+    @Override
+    public void receiveGeneratedData(byte[] data) {
+        // handle the incoming data as described in the benchmark description
+        String dataStr = new String(data);
+        logger.trace("receiveGeneratedData(" + new String(data) + "): " + dataStr);
+
+    }
+
+    @Override
+    public void receiveGeneratedTask(String taskId, byte[] data) {
+        // handle the incoming task and create a result
+        logger.debug("receiveGeneratedTask({})->{}", taskId, new String(data));
+
+        final String REGEX_SEPARATOR = ":\\*:";
+        String[] split = taskId.split(REGEX_SEPARATOR);
+        String urlTaskId = split[0];
+        String fileTrace = split[1];
+
+        String url = "http://localhost:8080/api/hobbitTask/" + urlTaskId;
```

Comment:
Is it really necessary to generate the URL again for every task and also create a new client? Shouldn't that be done once in the beginning to check whether the system is working. After that, the client can be used for all the tasks.

Additionally, I do not understand why the URL contains the task id... :thinking: